examples/07-Circuit/Virtual_Compliance.py

```diff
@@ -0,0 +1,258 @@
+"""
+Circuit: PCIE Virtual Compliance
+--------------------------------
+This example shows how you can generate a compliance report in PyAEDT using
+the new class Virtual Compliance.
```

Comment:
```suggestion
Circuit: PCIE virtual compliance
--------------------------------
This example shows how to generate a compliance report in PyAEDT using
the ``VirtualCompliance`` class.
```
Documentation should be timeless because what is new today isn't new tomorrow!